README.md

```diff
@@ -125,6 +125,44 @@ keycloak_failed_login_attempts{realm="test",provider="keycloak",error="invalid_u
 keycloak_failed_login_attempts{realm="test",provider="keycloak",error="user_not_found",client_id="application1"} 2.0
 ```
 
+##### keycloak_client_logins
+This counter counts every client login.  
```

Comment:
What exactly is a `client_login` and how does it differ from a normal login event (which already has a client label)?